app/controllers/geo/QueryHelpers.java

```diff
@@ -13,54 +13,61 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
+/**
+ * Helper methods for queries
+ */
+@SuppressWarnings("javadoc")
```

Comment:
Ignoring missing Javadocs is fine for test classes, but for production code the reason for the custom compiler check is to actually ensure all public API is documented, so we should add documentation here.